app/controllers/help_controller.rb

```diff
@@ -1,4 +1,8 @@
 class HelpController < ApplicationController
-	before_filter -> { @css_framework = :bootstrap }
-	layout ->(c) { request.format == :mobile ? "application" : "with_header_with_footer" }
-end
+  before_filter -> { @css_framework = :bootstrap }
+  layout -> (c) { request.format == :mobile ? "application" : "with_header_with_footer" }
+
+  def faq
+    gon.chat_enabled = AppConfig.chat.enabled?
```

Comment:
Weird... Thanks for the tip. Works no. What about the tags not being injected correctly ?

![injection](https://cloud.githubusercontent.com/assets/3429231/6316520/46232b88-ba29-11e4-829b-a4c11e8e0766.png)